pkg/http/api.go

```diff
@@ -1291,6 +1302,32 @@ func (a *api) onPublish(reqCtx *fasthttp.RequestCtx) {
 	}
 }
 
+func (a *api) getRetrySettingsFromRequest(reqCtx *fasthttp.RequestCtx) (string, int, int, error) {
+	var retryStrategy string
+	var retryMaxCount, retryIntervalInSeconds int
+	var retrySettingsError, err error
+	retryStrategy = strings.ToLower(string(reqCtx.Request.Header.Peek("retryStrategy")))
```

Comment:
Please, create a type for retryStrategy instead of just string.

Reply:
Fixed